parent/pom.xml

```diff
@@ -80,10 +80,10 @@
     <nexus-staging-maven-plugin.version>1.6.7</nexus-staging-maven-plugin.version>
     <osgi.symbolic.name>${project.artifactId}</osgi.symbolic.name>
     <resteasy-spring-boot-starter.version>2.3.0-RELEASE</resteasy-spring-boot-starter.version>
-    <slf4j.version>1.7.23</slf4j.version>
-    <spring.version>4.3.7.RELEASE</spring.version>
-    <spring-boot.version>1.5.2.RELEASE</spring-boot.version>
-    <swagger.version>1.5.19</swagger.version>
+    <slf4j.version>1.7.22.redhat-2</slf4j.version>
+    <spring.version>4.3.17.RELEASE</spring.version>
+    <spring-boot.version>1.5.13.RELEASE</spring-boot.version>
+    <swagger.version>1.5.17</swagger.version>
```

Comment:
IIRC we're very specific on the Swagger version we need, there is an issue that was fixed in 1.5.19 we rely on. In Syndesis we set the version to 1.5.19:

https://github.com/syndesisio/syndesis/blob/faeaa61ef038f62cbcf64e23d422c8cc864d564e/app/pom.xml#L65

Reply:
@zregvart I'll upgrade the swagger.version to 1.5.19 and rerun.